packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -397,8 +397,8 @@ export class AuthTransformer extends TransformerAuthBase implements TransformerA
       const subscriptionRoles = acm
         .getRolesPerOperation('listen')
         .map(role => this.roleMap.get(role)!)
-        // for subscriptions we only use static rules or owner rule
-        .filter(roleDef => roleDef.strategy === 'owner' || roleDef.static);
+        // for subscriptions we only use static rules or owner rule where the field is not a list
```

Comment:
There is a call to `removeAuthFieldsFromSubscriptionFilter`  method in line 276. Is that still necessary? [Refer](https://github.com/aws-amplify/amplify-category-api/pull/551/files#diff-0a38964c5525097a10b63bbbbea2f24278afce76a68e59353eaeb7d805372314R275)

Reply:
We are keeping the generated types but it would be harmless. We will completely revert the resolver code back.